packages/lisk-cryptography/test/nacl.js

```diff
@@ -0,0 +1,238 @@
+/*
```

Comment:
I think folder structure should be the same as the `src`.
In this case, `test/nacl/nacl.js`